torch/distributed/distributed_c10d.py

```diff
@@ -4394,7 +4394,7 @@ def _new_group_with_tag(
     global _world
 
     default_pg = _get_default_group()
-    device_id = default_pg.bound_device_id or None
+    device_id = default_pg.bound_device_id
```

Comment:
Interesting.. was this setting `device_id` to `None` when `bound_device_id` is `0`?